admin/src/graphql/searchUsers.js

```diff
@@ -5,15 +5,15 @@ export const searchUsers = gql`
     $searchText: String!
     $currentPage: Int
     $pageSize: Int
-    $notActivated: Boolean
-    $isDeleted: Boolean
+    $emailChecked: Boolean
```

Comment:
As this is a filter, I suggest to use `activated`

Reply:
i suggest `withActivated` or `withInActivated`